model/guitar.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const uuid = require('node-uuid');
+const debug = require('debug')('guitar:guitar');
+const createError = require('http-errors');
+const storage = require('../lib/storage.js');
+
+const Guitar = module.exports = function(name, type, make) {
+  if(!name) throw createError(400, 'bad request');
+  if(!type) throw createError(400, 'bad request');
+  if(!make) throw createError(400, 'bad request');
```

Comment:
You could make these a little more explicit.  Think of the end user and how an error could be more useful to them.